msc_pygeoapi/provider/cansips_rasterio.py

```diff
@@ -467,7 +467,7 @@ def get_file_list(
             month_files = [
                 str(file)
                 for file in Path(
-                    '/datasan/geomet/local/cansips-archives/100km/forecast/'
+                    '/data/geomet/local/cansips-archives/100km/forecast/'
                 ).rglob(filter)
```

Comment:
Unrelated, but `filter` is a Python keyword.  Suggest to rename to `filter_` as part of this PR.